.serena/memories/code_style_and_conventions.md

```
@@ -0,0 +1,74 @@
+
+      Raises:
+          ValueError: When something is invalid
+      """
+  ```
+
+## Imports
+- Organize: standard library → third-party → local imports
+- One import per line for clarity
+- Sort alphabetically (pre-commit will enforce via ruff)
+
+## Neo4j Integration Pattern
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)